documentation/docs/11-ssr-and-javascript.md

```diff
@@ -64,14 +64,15 @@ In many cases, you'll only want to prerender specific pages in your app. You'll
 
 The prerenderer will start at the root of your app and generate HTML for any prerenderable pages it finds. Each page is scanned for `<a>` elements that point to other pages that are candidates for prerendering — because of this, you generally don't need to specify which pages should be accessed. If you _do_ need to specify which pages should be accessed by the prerenderer, you can do so with the `pages` option in the [prerender configuration](#configuration-prerender).
 
+The primary benefit of fully prerendering a site is that you do not need to maintain / pay for NodeJS servers to perform SSR to serve pages. The SSR happens at deploy time, after which the site can be served from CDNs, leading to great performance. This delivery model is so powerful that it has a nickname: JAMStack. While static sites are a clear candidate for this model, JAMStack sites are not limited to static content. You can build fully authenticaled, dynamic, personalized sites with this model, just make sure all user specific data is fetched and rendered client side. Parts of pages may be pre-rendered, possibly with some placeholders, then replaced on the client with user specific data fetched from API endpoints in onMount(). You need API servers, instead of page servers.
```

Comment:
'vs just doing ssr' - that is the core of this change.

The API servers are backend servers, running Python, Java, Ruby, Node or whatever.
They can be your own, or increasingly services like Auth0, Stripe etc.
All dynamic behavior is between those backend servers and Javascript on the client, no need for a middle SSR tier.

Reply:
Maybe instead of "You need API servers, instead of page servers" we could say something about the pros and cons like "This can reduce hosting costs by removing the need to do server-side rendering, but also means that changes to your site may be picked up by search engines more slowly."

Reply:
> The SSR happens at deploy time, after which the site can be served from CDNs, leading to great performance. This delivery model is so powerful that it has a nickname: JAMStack

I'd prefer to make this sound a bit more neutral and sounding less like we're pushing any one particular solution. I'd rather discuss the pros and cons and let the user decide if it's the best fit for them. We could just remove the more flowery language to say:

>The SSR happens at deploy time, after which the site can be served from CDNs. This delivery model has a nickname: JAMStack

Part of the reason I suggest this is that it's often a good fit, but may not always be. While it will usually improve time-to-first byte, it can make time-to-interactive and other metrics further down the processing pipeline worse especially for more complicated pages. Instead of making a single round trip over the internet to the backend server, you are more likely to have to make multiple round trips over the internet to render the page

Reply:
I don't think the language is suggesting 'this is the way'.
It is just trying to explain characteristics of this particular way.